thaomonster/network-request
# Feature/html css

### What does this PR do? 
>Clearly and concisely describe the feature. 
This build out the comp in html & css.

### Is this a feature or refactor?  
feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.
N/A

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
index.html & styles.css beginning to end

### How should this be tested?
>What steps need to be taken to confirm functionality? 
In your terminal open index.html

### Steps To Take
>Any questions and possible steps moving forward.
Connecting the API
